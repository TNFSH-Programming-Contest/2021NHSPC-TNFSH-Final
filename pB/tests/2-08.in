8 9 7
7 6 3 5 1 8
1 6 7 5 9 2
8 6 1 2 3 7
3 9 7 2 4 5
2 4 7 9 1 8
5 3 9 1 4 6
9 6 1 2 4 3
8 3 6 4 1 7
7 3 2 8 1 7 5 6
5 2 7 5 4 3
6 3 8 1 7 4 6
5 4 7 8 5 6
4 4 1 2 6
6 3 2 1 8 6 7
6 1 2 3 8 5 4
4 5 3 8 1
5 7 4 6 5 2
