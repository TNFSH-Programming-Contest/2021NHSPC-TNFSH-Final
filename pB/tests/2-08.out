1 2
1 5
1 4
0
1 6
0
1 1
2 3 8
1 7
